clients/app/src/utils/helpers/dummy.ts

```diff
@@ -262,7 +249,7 @@ const generateMatch1 = () => {
     leagueImageURI: generateLeagueURL(),
     matchState: MatcheTimeState.ExtraTime,
     startTime: randomDate(new Date(2021, 10, 10), new Date(2021, 11, 11)),
-    venue: generateSupportersClubName(),
+    venue: 'aa',
```

Comment:
what is this ?

Reply:
this file should be removed very soon.

Reply:
oh I see, sorry didn't realized this is a dummy file